src/main/resources/application.yml

```diff
@@ -1,3 +1,18 @@
+#Spring boot configuration
 spring:
+  #JPA properties
   jpa:
-    show-sql: true
\ No newline at end of file
+    show-sql: true
+    open-in-view: false
+
+#Server configuration
+server:
+  port: 8080 #set your port
+  error:
+    include-message: always
+
+# Logger configuration
+logging:
+  level:
+    org.springframework: info
+    org.hibernate: info
```

Comment:
Would `root` be able to substitute both of these lines?

`logging.level.root: info`

Reply:
I think it's better to specifry, as you will not always need the same level on both. The port is also 8080 by default, but I think it's better to have the option there so it's explicitly changeable.

Reply:
LGTM